inc/namespace.php

```diff
@@ -31,3 +33,42 @@ function on_plugins_loaded() {
 		require_once ROOT_DIR . '/vendor/johnbillion/query-monitor/query-monitor.php';
 	}
 }
+
+function qm_file_path_map( $map ) : array {
```

Comment:
Needs a phpdoc block.

Reply:
I think this should actually probably move into the specific server modules, as we don't need/want this at all on production.

Reply:
agreed, here's  PR for local-chassis https://github.com/humanmade/altis-local-chassis/pull/39

Reply:
and local-server https://github.com/humanmade/altis-local-server/pull/56